### ISSUE TITLE
Don't include "autoload-dev" classMaps from vendor packages

### ISSUE THREAD

**Issue by d8vjork** [DEVSENSE/phptools-docs#220]
### Current behaviour

Class autocompletion is filled with classes that shouldn't be loaded in the backend via Composer but PHPTools recognise them anyways:

<img width="1031" alt="image" src="https://user-images.githubusercontent.com/2331052/207663608-8a5ecb2d-e1a4-43cf-9296-1379eef4b3e4.png">

### Expected behaviour

Only own package dependencies under `autoload-dev` (composer.json) should be included on the autocompletion, but not the rest coming from other packages

**Comment by jakubmisek** [comment]
good point - we have to prioritize the list, and also do not show items that are not auto-loadable ... It's on our to-do list.
